index.js

```diff
@@ -0,0 +1,167 @@
+// FEATURED SECTION
+
+const container = document.querySelector('#featured');
+
+const features = [
+  {
+    thumbnail: 'images/first.jpg',
+    name: 'Osandi Robinson',
+    occupation: 'Designer in Charlotte, NC, United States',
+    description:
+      'Mentored at Apple, Osandi has spent over ten years helping teams.',
+  },
+  {
+    thumbnail: 'images/second.jpg',
+    name: 'Danielle Thompson',
+    occupation: 'Designer in Singapore, Singapore',
+    description:
+      'Danielle is a seasoned UI/UX product designer specializing in the sports and fintech market.',
+  },
+  {
+    thumbnail: 'images/third.jpg',
+    name: 'Mike Slone',
+    occupation: 'Designer in Ladera Ranch, CA, United States',
+    description:
+      'Mike is a highly experienced and successful product, UX, UI, visual, and branding designer.',
+  },
+  {
+    thumbnail: 'images/fourth.jpg',
+    name: 'Amélie Lamont',
+    occupation: 'Designer in New York, NY, United States',
+    description:
+      'Amélie is an independent product designer, writer, and UX teacher based in New York.',
+  },
+  {
+    thumbnail: 'images/fivth.jpg',
+    name: 'Lee Hsieh',
+    occupation: 'UX/UI Designer in Tokyo, Japan',
+    description:
+      'Lee has designed everything from sex education software for kids to flight management apps for pilots.',
+  },
+  {
+    thumbnail: 'images/sixth.jpg',
+    name: 'Greg Kujda',
+    occupation: 'Designer in Long Beach, CA, United States',
+    description:
+      'Greg is a hybrid storyteller with deep experience leading UI/UX, visual, and motion design.',
+  },
+];
+
+// CREATING FEATURED SECTION FROM ARRAY DATA AND POPULATING DYNAMICALLY
+
+const populate = function (number) {
+  function generate(items) {
+    items.forEach((feature) => {
+      const content = ` <div class="featured-items">
+                          <div class="lefts">
+                            <img src=${feature.thumbnail} alt="Tutor-image" />
+                          </div>
+                          <div class="rights">
```

Comment:
- [ ] (Optional) Your design is one of the best I have seen for this capstone. It would be nice to make it better by making the image placeholder for each card look like the one used in the template design provided (the chessboard background). Remember that your project should look very identical to the template provided by Behance 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/157916038-42992823-8903-46c4-8c1b-30a7c0b01ac1.png)

### Behance Design

![image](https://user-images.githubusercontent.com/77106962/157916132-0a0a5367-357b-4e05-a911-399952b2d6bf.png)